datadog_lambda/__init__.py

```diff
@@ -12,14 +12,7 @@
 
 # The minor version corresponds to the Lambda layer version.
 # E.g.,, version 0.5.0 gets packaged into layer version 5.
-try:
-    import importlib.metadata as importlib_metadata
-except ModuleNotFoundError:
-    import importlib_metadata
-
-__version__ = importlib_metadata.version(__name__)
-
-
+from datadog_lambda.version import __version__  # noqa: E402
```

Comment:
Also, it looks like the linter is complaining on the GitHub side, since we are not using the variable, why might that be?

Reply:
I added the `noqa: F401` comment to this line as well, should fix the linting issue.